src/main/resources/com/google/api/codegen/nodejs/sample.snip

```diff
@@ -62,10 +62,14 @@
     @end
     @if class.hasRequestBody
       resource: {
-        @if class.methodVerb == "PATCH"
-          // TODO: Add desired properties of the request body. Only added properties
+        @switch class.methodVerb
+        @case "PATCH"
+          // TODO: Add desired properties of the request body. Only these properties
```

Comment:
This inconsistency is deliberate, because in Node.js/Python one is constructing an object literal/dictionary whose collection of properties/entries is not predetermined, hence these are "added" rather than "assigned". I changed the wording here because "added properties" seemed open to misinterpretation (could mean added to the original object rather than added to the request body).